src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs

```diff
@@ -361,9 +361,10 @@ internal PSRemotingDataStructureException(Exception innerException, string resou
         /// </summary>
         /// <param name="info"></param>
         /// <param name="context"></param>
+        [Obsolete("Legacy serialization support is deprecated since .NET 8", DiagnosticId = "SYSLIB0051")] 
```

Comment:
The `[Serializable]` attribute was not removed from the class.

Reply:
removed